README.md

```diff
@@ -11,6 +11,16 @@ This is not meant to generate data for your tests, if this is what
 you need, checkout [ecto_fixtures](https://github.com/dockyard/ecto_fixtures) or
 [ex_machina](https://github.com/thoughtbot/ex_machina) instead.
 
+## Compatible versions
+
+Version `0.3.0` or lower:
+
+* Elixir ~> `1.11.x` or lower.
+
+Version `1.0.0` or higher:
```

Comment:
@CristhianRodriguezMolina since this library does not have a stable version (1.0.0) we're forced to use a lower version `0.4.0`